### PR TITLE
Added loading animation

### PR DESCRIPTION
**Pull request by Nandan328**

### CODE REVIEW

**Comment by CharanMN7** [comment]
Tasks for the reviewers:
- [ ] Check for responsiveness across different device sizes
- [ ] Double check build local status

File: components/ui/loading.tsx
Comment (CharanMN7) [inline components/ui/loading.tsx:1]: Please change the file name and the component name to a more descriptive name like "LoadingSpinner".

**Review by CharanMN7 — CHANGES_REQUESTED** [review]

File: components/ui/loading.tsx
Comment (Nandan328) [reply components/ui/loading.tsx:1]: Thanks for pointing that out!
I've changed the files name and component as you suggested.

Comment (CharanMN7) [reply components/ui/loading.tsx:1]: Looking good!

**Review by CharanMN7 — APPROVED** [review]

**Comment by CharanMN7** [comment]
The loading spiner looks great! Thank you for your contribution @Nandan328 !

**Pull request merged**